### ISSUE TITLE
1:30 Voiceover Audio

### ISSUE THREAD

**Issue by CollinEnstad** [CollinEnstad/Bitcoin-Cash-Barbershop-Spot#2]

**Comment by CollinEnstad** [comment]
https://drive.google.com/file/d/1h3tXitGCK9jtWKqTzVgU8_dPlr6Ozu1O/view?usp=sharing
https://drive.google.com/file/d/1lVNGZajYraYHNWivAOsePcIBtlavpX4Z/view?usp=sharing

**Comment by Solon-Archon** [comment]
After listening to it a few times more. I think the easiest way to cut the time down is to remove the 'mouthing' sections. They break up the pace and don't really add anything. The script works without any mouthing. It's also much easier to shoot. Thoughts?

**Comment by CollinEnstad** [comment]
Yeah I agree - check the 30 second edit, it flows pretty darn nicely without all the fluff added to it
